src/Common/src/System/Collections/HashHelpers.cs

```diff
@@ -41,23 +39,12 @@ internal static class HashHelpers
             1103, 1327, 1597, 1931, 2333, 2801, 3371, 4049, 4861, 5839, 7013, 8419, 10103, 12143, 14591,
             17519, 21023, 25229, 30293, 36353, 43627, 52361, 62851, 75431, 90523, 108631, 130363, 156437,
             187751, 225307, 270371, 324449, 389357, 467237, 560689, 672827, 807403, 968897, 1162687, 1395263,
-            1674319, 2009191, 2411033, 2893249, 3471899, 4166287, 4999559, 5999471, 7199369 };
-
-        public static bool IsPrime(int candidate)
-        {
-            if ((candidate & 1) != 0)
-            {
-                int limit = (int)Math.Sqrt(candidate);
-                for (int divisor = 3; divisor <= limit; divisor += 2)
-                {
-                    if ((candidate % divisor) == 0)
-                        return false;
-                }
-                return true;
-            }
-            return (candidate == 2);
-        }
-
+            1674319, 2009191, 2411033, 2893249, 3471899, 4166287, 4999559, 5999471, 7199369, 8639249, 10367101,
```

Comment:
This is going to change allocations in certain cases.  For example, before your change, adding 8M items to a ```HashSet<int>``` would result in the _slots and _buckets arrays ending at 11998949 in length, and after your change they'll be 12440537 in length (~400K bigger each).  Whether that's good or bad for the scenarios that hit this, I don't know.  I do like that we can simplify the code by expanding out the table, but we should understand the full ramifications of doing so before such a change is made.